pt-BR/mail/chrome/messenger/phishing-afterload-warning-message.dtd

```diff
@@ -1,28 +1,28 @@
-<!ENTITY safeb.palm.warning.title "Suspected Web Forgery">
+<!ENTITY safeb.palm.warning.title "Suspeita de falsificação na Internet">
 
-<!ENTITY safeb.palm.message.p1 "This page has been reported as a web forgery designed to trick users into sharing personal or financial information. Entering any personal information on this page may result in identity theft or other fraud. &#160;">
-<!ENTITY safeb.palm.message.p1.linkText "Read more &#187;">
-<!ENTITY safeb.palm.p1.linkStatusText "Read more &#133;">
+<!ENTITY safeb.palm.message.p1 "Esta página foi reportada como falsificação projetada para enganar os usuários e fazer com que forneçam dados pessoais ou informações financeiras. Digitar informações pessoais nesta página pode resultar em roubo de identidade ou outros tipos de fraude. &#160;">
+<!ENTITY safeb.palm.message.p1.linkText "Leia mais em &#187;">
+<!ENTITY safeb.palm.p1.linkStatusText "Leia mais em &#133;">
 
-<!ENTITY safeb.palm.message.p2.start "These types of web forgeries are used in scams known as phishing attacks, in which fraudulent web pages and emails are used to imitate sources you may trust. You can find out more about ">
-<!ENTITY safeb.palm.message.p2.linkText "how &brandShortName; protects you">
-<!ENTITY safeb.palm.message.p2.end " from phishing attacks.">
+<!ENTITY safeb.palm.message.p2.start "Esses tipos de falsificações são utilizados em golpes conhecidos como ataques de phishing, em que páginas de Internet falsas e e-mails são utilizados para imitar fontes nas quais você confia. Você pode saber mais sobre ">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY safeb.palm.message.p2.start "Esses tipos de falsificações são utilizadas em golpes conhecidos como ataques de phishing, em que páginas de Internet falsas e mensagens de correio eletrônico são utilizados para imitar fontes nas quais você confia. Você pode saber mais sobre ">